src/pages/index.tsx

```diff
@@ -8,11 +8,14 @@ import React from 'react';
 import HomepageFeatures from '../components/homepage-features';
 import HomepageUsage from '../components/homepage-usage';
 import HomepageUsers from '../components/homepage-users';
+import { useAds } from '../hooks/use-ads';
 import css from './index.module.scss';
 
 function HomepageHeader() {
   const { siteConfig } = useDocusaurusContext();
 
+  useAds({ selector: '.ads-container' });
```

Comment:
Can you document here that on the homepage (and only on the homepage), ads are added in the footer?

We could consider adding ads in the footer of all non-documentation pages (such as /release)

Reply:
done